shell/platform/tizen/flutter_tizen_engine.h

```diff
@@ -11,6 +11,7 @@
 #include "flutter/shell/platform/common/client_wrapper/include/flutter/plugin_registrar.h"
 #include "flutter/shell/platform/common/incoming_message_dispatcher.h"
 #include "flutter/shell/platform/embedder/embedder.h"
+#include "flutter/shell/platform/tizen/channels/app_control_channel.h"
```

Comment:
```suggestion
#ifndef __X64_SHELL__
#include "flutter/shell/platform/tizen/channels/app_control_channel.h"
#endif
```
You should mask anything related to app_control in `flutter_tizen_engine.h/cc` and `flutter_tizen.cc` or otherwise create a stub file named `app_control_channel_stub.cc`.

Reply:
@swift-kim could you please give me a little background for such masking? I would like to understand why it is needed.

Reply:
@bwikbs has added support for Linux EFL desktop to our embedder in https://github.com/flutter-tizen/engine/pull/125. AppControl is a Tizen stuff, and is incompatible with Linux desktop, so anything related to AppControl should not be included when building the desktop embedder.

https://github.com/flutter-tizen/engine/pull/147#issue-691252272 has some explanation about filename suffixes. But I think making some use of `#ifdef` is enough in this specific case.

Reply:
Thank you for sharing the context. I think that I've applied all suggestions.